compiler/src/iree/compiler/Dialect/LinalgExt/IR/LinalgExtOps.cpp

```diff
@@ -1885,6 +1885,64 @@ CustomOp::reifyResultShapes(OpBuilder &builder,
   return success();
 }
 
+/// End `ReifyRankedShapedTypeOpInterface` implementation
+
+/// Start `AggregateOpInterface` implementation
+FailureOr<SmallVector<Value>> CustomOp::decomposeOperation(OpBuilder &builder) {
```

Comment:
wait, this should be moved to `AggregatedOpInterfaceImpl.cpp`. Should we also move the file to `IR/`?

https://github.com/iree-org/iree/blob/main/compiler/src/iree/compiler/Dialect/LinalgExt/Transforms/AggregatedOpInterfaceImpl.cpp